src/constants.js

```diff
@@ -57,6 +57,7 @@ export const ReinhardToneMapping = 2;
 export const CineonToneMapping = 3;
 export const ACESFilmicToneMapping = 4;
 export const CustomToneMapping = 5;
+export const AgXToneMapping = 6;
```

Comment:
```suggestion
export const AgXToneMapping = 99;
```

Reply:
If this is going to change lets do it in another PR - sounds like there are concerns about backwards compatibility.